SayTheirNames/Source/Controller/Home/Person/PersonController.swift

```diff
@@ -254,11 +253,11 @@ extension PersonController: UITableViewDataSource {
             return infoCell
         case .story:
             let storyCell = cell as! PersonOverviewTableViewCell
-             storyCell.setupCell(title: "THEIR STORY", description: person.bio)
+            storyCell.setupCell(title: L10n.Person.theirStory.uppercased(), description: person.bio)
             return storyCell
         case .outcome:
             let overviewCell = cell as! PersonOverviewTableViewCell
-            overviewCell.setupCell(title: "OUTCOME", description: person.context)
+            overviewCell.setupCell(title: L10n.Person.outcome.uppercased(), description: person.context)
```

Comment:
@grysvn Can we invoke `uppercase()` inside the `func:setupCell(title:_)` to make it a bit neat

Reply:
And resolve the conflicts as well :))

Reply:
Good call-out! Addressed in latest commit.